x/stake/handler.go

```diff
@@ -187,8 +176,12 @@ func BondCoins(ctx sdk.Context, k Keeper, bond DelegatorBond, candidate Candidat
 	if err != nil {
 		return err
 	}
-	newShares := k.candidateAddTokens(ctx, candidate, amount.Amount)
+	p := k.GetPool(ctx)
+	var newShares sdk.Rat
```

Comment:
nit: := is probably cleaner here

Reply:
oh okay yeah can do that here actually. 👍